applications/vanilla/models/class.discussionmodel.php

```diff
@@ -3236,4 +3237,49 @@ public function structuredData(array $discussion): array {
         }
         return $result;
     }
+    /**
+     * Returns array of translations for writeDiscussion widget
+     *
+     * @return array
+     */
+    protected static function tInit() {
+        return [
+            '%s view' => Gdn::translate('%s view'),
+            '%s views' => Gdn::translate('%s views'),
+            '%s comment' => Gdn::translate('%s comment'),
+            '%s comments' => Gdn::translate('%s comments'),
+            '%s point' => Gdn::translate('%s point'),
+            '%s points' => Gdn::translate('%s points'),
+            'Most recent by %1$s' => Gdn::translate('Most recent by %1$s'),
+            'Started by %1$s' => Gdn::translate('Started by %1$s'),
+            'via %s' => Gdn::translate('via %s'),
+            'Options' => Gdn::translate('Options')
+        ];
+    }
+
+    /**
+     * bigPlural global function replacement
+     *
+     * @param int $number
+     * @param string $format
+     *
+     * @return string
+     */
+    public static function bigPluralHtmlSpan(int $number, string $format) {
+        if ($number === 1) {
```

Comment:
I understand because of the static nature of the caching here we have to keep something in this class, but I'd prefer to see the logic of these functions in a common place so we don't continue to re-implement it.

Reply:
| I am also of the opinion that bigPluralHtmlSpan and pluralTranslate are not functions/methods that are specific to discussions.
I do agree that these 2 functions are not part of DiscussionModel, but imho these both functions should never exist at all. To me that is temporary solution to copy-paste it here to keep same functionality but not to pollute to other classes. I can move these 2 functions to any other better candidate you would chose. But please point any.

Reply:
@charrondev mentioned `TranslationUtils` and that seems fine with me!